packages/unmock-core/src/service/interfaces.ts

```diff
@@ -106,6 +109,16 @@ export interface IService {
    */
   readonly schema: OpenAPIObject;
 
+  /**
+   * Holds the absolute path where the service specification resides.
+   */
+  readonly absPath: string;
+
+  /**
+   * Used to dereference items in the schema when needed
+   */
+  readonly dereferencer: Dereferencer;
```

Comment:
Do we expect different kinds of dereferencers instead of a single function?

Reply:
Why does this need to be exposed by the service? 🤔

Reply:
It's used with the generator - hence the public attribute.
Not expecting different kinds of dereferencers for now. It mostly makes it easier to mock. We could expand on this though (see my other replies).